irohad/ametsuchi/temporary_wsv.hpp

```diff
@@ -39,6 +39,19 @@ namespace iroha {
      */
     class TemporaryWsv {
      public:
+      /**
+       * Wrapper for savepoints in wsv state; rollbacks to savepoint, if
+       * destroyed without explicit release, releases it otherwise
+       */
+      struct SavepointWrapper {
+        /**
+         * Release the savepoint
+         */
+        virtual void release() = 0;
```

Comment:
Can we put release logic into destructor?

Reply:
I don't think that's a good idea. It completely implementation specific stuff

Reply:
Yeah, we've already spoken about this with Kamil, decided to leave it as-is